build/dsc/resource.rb

```diff
@@ -16,6 +16,7 @@ def initialize(mof_class, mof_path)
       @embedded_properties = nil
       @dsc_module          = nil
       @ps_module           = nil
+      @absentable          = nil
```

Comment:
That's about setting a default so we can do a simple comparison to see if it's been calculated already (as we query `absentable` quite a bit when building templates)